svf/include/SVFIR/SVFValue.h

```diff
@@ -320,6 +320,15 @@ class SVFFunction : public SVFValue
     std::vector<const SVFBasicBlock*> allBBs;   /// all BasicBlocks of this function
     std::vector<const SVFArgument*> allArgs;    /// all formal arguments of this function
     std::vector<std::string> annotations; /// annotations of this function
+    SVFBasicBlock *entryBlock;                  /// entry BasicBlock of this function
```

Comment:
I think we don't need an entryBlock because it is always the first block in allBBs. This is consistent with LLVM's API called getEntryBlock(), which simply returns the front of its basic blocks.

Reply:
Thank you for your review. Yeah, here is redundant.